source/NsDepCop.Analyzer/RoslynAnalyzer/DiagnosticDefinitions.cs

```diff
@@ -16,7 +16,7 @@ public static class DiagnosticDefinitions
         public static readonly DiagnosticDescriptor IllegalDependency = CreateDiagnosticDescriptor(
             "NSDEPCOP01",
             "Illegal namespace reference.",
-            "Illegal namespace reference: {0}->{1} (Type: {2}->{3})",
+            "Illegal namespace reference: {0}->{1} (Type: {2}->{3}) Allowed types are: {4}",
```

Comment:
I realise that we'll need two versions of this message, one with and one without allowed types.

Reply:
Resolved